dts/bindings/mtd/atmel,at45.yaml

```diff
@@ -45,6 +45,18 @@ properties:
     required: true
     description: Flash page size in bytes.
 
+  workaround-broken-read:
+    type: boolean
+    description: |
+      Some AT45 flash chips do not support the regular read opcode. If set, the driver will perform
```

Comment:
Mention the code, as you did for the replacement below.

Reply:
updated the description